miden-lib/asm/kernels/transaction/lib/tx.masm

```diff
@@ -189,14 +189,14 @@ end
 #!
 #! The input block_height_delta is added to the block reference number in order to output an upper
 #! limit at which the transaction will be considered valid (not expired).
-#! This value can be later decreased, but not increased. 
+#! This value can be later decreased, but not increased.
 #!
 #! Inputs: [block_height_delta, ...]
 #! Output: [...]
 #!
 #! Where:
 #! - block_height_delta is the desired expiration time delta (1 to 0xFFFF).
-export.update_expiration_block_num
+export.update_expiration_delta
```

Comment:
I think we [want to keep this name the same as it was](https://github.com/0xPolygonMiden/miden-base/pull/897#discussion_r1779107516) although I don't mind this option too much either.

Reply:
I would keep the name the same here.